### PR TITLE
Beginning usage policy page

### PR DESCRIPTION
**Pull request by jules32**
Hi @ateucher , 

I started a page for Policies & Usage Costs; this can eventually become a section with subchapters if that makes most sense but it now is a start. This can have our policies and examples of cost and recommendations. 

This is the start of #229

### CODE REVIEW

**Comment by ateucher** [comment]
This looks like a great start Julie. Do you want us to keep working on this branch and merge when done, or are you looking for a review now to merge what you've got so far?

**Comment by jules32** [comment]
Great question. Would you merge it to give it more visibility and we can continue to add to it and shape it from there?

File: _quarto.yml [hunk 1]
```diff
@@ -118,6 +118,8 @@ website:
         contents:
           - text: "Workshop Setup"
             href: workshops/setup.md
+      - text: "Policies & Usage Costs" 
+        href: policies-usages/index.md
```
Comment (ateucher) [inline _quarto.yml:122]: There is no "s" in the directory name

```suggestion
        href: policies-usage/index.md              
```

**Review by ateucher — APPROVED** [review]
Looks great! Just one small mismatch in file path names which results in a 404 in the rendered book

**Pull request merged**